app/questions.py

```diff
@@ -1,71 +1,87 @@
+"""this module has routes for questions and answers"""
+
 from flask_restplus import Resource
 from flask_restplus import reqparse
 from app.models import Models
 from utils.validator import Validate as validate
 
 
 class Questions(Resource):
+    """class for question routes"""
```

Comment:
This docstring is too brief, it should at least capture the methods in the class